code-challenges.test.js

```diff
@@ -14,33 +14,153 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+// RED:
+
+describe("testPeople", () => {
+  it("returns an array with name capitalized in a sentence", () =>{
+    const people = [
+      { name: "ford prefect", occupation: "a hitchhiker" },
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" }
+    ]
+    // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+    expect(testPeople(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
+// FAIL  ./code-challenges.test.js
+// testPeople
+//   ✕ returns an array with name capitalized in a sentence
+
+// ● testPeople › returns an array with name capitalized in a sentence
+
+//   ReferenceError: testPeople is not defined
 
 // b) Create the function that makes the test pass.
 
+// GREEN:
+
+// PSUEDOCODE:
+
+// declare a function testPeople
+// input: array of objects
+// output: array of strings with first letter capitalized
+// Process: create a variable capital1 which will take zero index of the value name and use map to iterate through the of the array to capitalize the first letter. create a variable capital2 and take key name and split at the space then map through the strings. Uppercase at the zero index of the string plus slice and join back together. return the sentence by utilizing string interpolation.
+
+const testPeople = (array) => {
+  let capital1 = array.map(value => {
+    let capital2 = value.name.split(" ").map(string => 
+    string[0].toUpperCase() + string.slice(1)).join(" ")
+    return `${capital2} is ${value.occupation}`
+})
+return capital1
+}
+// *** Note: Not sure why it is not passing. The expected and the received are identical in the terminal. ***
```

Comment:
I am not too sure either but I would look into why expected is - (as if it is being taken away, and the received + is showing it as adding characters ...)